ldk/javascript/examples/self-test-loop/src/tests/document/index.tsx

```diff
@@ -0,0 +1,112 @@
+/* eslint-disable no-async-promise-executor */
+import { React, ReactWhisper, document, network, whisper } from '@oliveai/ldk';
+import { Workbook, PDFContentType, PDFOutput } from '@oliveai/ldk/dist/document/types';
+
+export const testDocumentEncodeAndDecode = (): Promise<boolean> =>
+  new Promise(async (resolve, reject) => {
+    const workbook: Workbook = {
+      worksheets: [
+        {
+          hidden: false,
+          hiddenColumns: [],
+          hiddenRows: [],
+          name: 'name',
+          rows: [{ cells: [{ value: 'value' }] }],
```

Comment:
If this were expanded out it would be much easier to read! But the way you have it now seems consistent with similar lines elsewhere ;9

Reply:
@mimoduo Well I believe I wrote this test before and Josef moved this test from ts file to tsx file. 😂  I think Prettier makes it look this way. If there's more cells & values, it will be expanded.